scripts/fast-reboot

```diff
@@ -106,6 +106,18 @@ function clear_warm_boot()
     fi
 }
 
+function init_warm_reboot_states()
+{
+    # If the current running instanace was booted up with warm reboot. Then
+    # the current DB contents will likely mark warm reboot is done.
+    # Clear these states so that the next boot up image won't get confused.
+    if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
+        for key in `/usr/bin/redis-cli -n 6 keys "WARM_RESTART_TABLE|*"`; do
+            /usr/bin/redis-cli -n 6 hdel $key "state" > /dev/null
```

Comment:
I don't remember WARM_RESTART_TABLE in stateDB is saved during warm reboot, how will it affect next warm reboot?
But this change is good by itself.

Reply:
It is saved so that the counters can keep on increasing. Anything didn't get explicitly flushed will be saved.

Reply:
Would it make sense to keep the counter?

Reply:
I see, only state field gets deleted.

Reply:
Yes :-)